google/cloud/datastore/batch.py

```diff
@@ -290,7 +290,7 @@ def _assign_entity_to_pb(entity_pb, entity):
 
     Helper method for ``Batch.put``.
 
-    :type entity_pb: :class:`google.cloud.datastore._generated.entity_pb2.Entity`
+    :type entity_pb: :class:`.datastore._generated.entity_pb2.Entity`
```

Comment:
The `.` just says "start here" and find the only match. It's not a relative path.

Also the `._generated` stuff won't show up any way so it's just good habits to link to, with no tangible result (other than me staying under 80 chars here).